src/main/resources/languages/message_fr.properties

```diff
@@ -6325,8 +6325,8 @@ siteInfo.instruction.patient.nationality = S\u00E9lectionner "vrai" pour \
                                            nationalit\u00E9 des patients soit \
                                            entr\u00E9 dans la base de \
                                            donn\u00E9es.
-siteInfo.lab.director                    = Le nom du directeur du laboratoire \
-                                           sur les rapports
+siteInfo.lab.director                    = Nom du responsable de laboratoire \ 
+                                           pour l’en-tête du rapport
```

Comment:
Special characters often get corrupted as they move from system to system.

![image](https://user-images.githubusercontent.com/10550869/169906008-fd011265-6158-4ce6-81f5-1acdec79a4ce.png)


 It is recommended to include non-ascii characters as encoded versions. 
 ’ = ' 
 ê = \u00EA